src/constants.js

```diff
@@ -0,0 +1,7 @@
+export const usersUrl = 'https://api.github.com/users';
```

Comment:
Which constants do you mean? Change the filename appropriately

Reply:
constants
  -ApiStatus
  -Url